code/obj/item/instruments.dm

```diff
@@ -75,6 +75,8 @@
 			if (src.dog_bark)
 				for_by_tcl(G, /obj/critter/dog/george)
 					if (IN_RANGE(G, T, 6) && prob(60))
+						if(ON_COOLDOWN(G, "george howl", 10 SECONDS))
+							return
```

Comment:
This skips `post_play_effect` and also makes the proc not return `1` as line 84 wants, me no like.
```suggestion
							continue
```

Reply:
I am a dumb dumb who did not notice there was code after.